pkgs/applications/video/showmethekey/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  glib,
+  meson,
+  ninja,
+  libevdev,
+  json-glib,
+  cairo,
+  pango,
+  polkit,
+  libinput,
+  gtk3,
+  gtk4,
+  wrapGAppsHook,
+  libxkbcommon,
+  pkg-config,
+}:
+stdenv.mkDerivation rec {
+  pname = "showmethekey";
+  version = "1.7.3";
+
+  src = fetchFromGitHub {
+    owner = "AlynxZhou";
+    repo = "showmethekey";
+    rev = "v${version}";
+    sha256 = "sha256-hq4X4dG25YauMjsNXC6Flco9pEpVj3EM2JiFWbRrPaA=";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [
+    glib
+    meson
+    ninja
+    cmake
+    cairo
+    pango
+    json-glib
+    pkg-config
+    libevdev
+    libinput
+    libxkbcommon
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    polkit
+    gtk3
+    gtk4
```

Comment:
Are two versions of GTK necessary? Seems odd.

Reply:
Forgot to remove gtk3. sorry